templates/shibboleth2.xml.j2

```diff
@@ -12,16 +12,14 @@ https://wiki.shibboleth.net/confluence/display/SHIB2/NativeSPConfiguration
     clockSkew="1800">
 
     <!-- FIXME: change the entityID to your hostname. -->
-    <ApplicationDefaults entityID="https://{{ servername }}/sp"
-        REMOTE_USER="eppn" attributePrefix="AJP_">
+    <ApplicationDefaults entityID="https://{{ dataverse.glassfish.siteurl }}/sp"
```

Comment:
will the protocol be duplicated here?

Reply:
I do not think so.

Reply:
siteurl is offered as a group_var for setting the glassfish siteurl jvm-option, which is required for DOI publishing. if not set as a group_var, the FQDN task block constructs it from a best-guess. siteurl will include the protocol (http:// or https://...) so I'm afraid including https:// in the string above will break the Shib config.

Reply:
dataverse.glassfish.siteurl is not given value anywhere in tasks, please check again!

Reply:
OK, I see that at other places, it implicitly assumes that dataverse.glassfish.siteurl is a full URL. I will change the code accordingly.